web/src/utils/transformDataHelper.tsx

```diff
@@ -94,10 +94,54 @@ export function getProfileContacts(contactSet: any[]): object {
   contactSet.forEach((contact: any) => {
     if (contact.roleId === 1) {
       contacts.POEmail = contact.email;
+      contacts.POName = contact.firstName + ' ' + contact.lastName;
+      contacts.POGithubId = contact.githubId;
     }
     if (contact.roleId === 2) {
       contacts.TCEmail = contact.email;
+      contacts.TCName = contact.firstName + ' ' + contact.lastName;
+      contacts.TCGithubId = contact.githubId;
     }
   });
   return contacts;
-};
\ No newline at end of file
+};
+
+// convert datetime string from YYYY-MM-DDTHH:MM:SSZ to DD-MM-YYYY HH:MM
+function convertDatetime(ISODatetimeString: string): string {
+  const splitted = ISODatetimeString.split('T');
```

Comment:
The package `moment` is super handy at manipulating and managing time/dates.

Reply:
Yes, I was planning to adapt moment.js but came across this in their docs "We recognize that many existing projects may continue to use Moment, but we would like to discourage Moment from being used in new projects going forward...it is now a legacy project in maintenance mode".

Reply:
Interesting. Good catch. If its in maintenance mode then ya, let's not use it. I'll read the docs to see what they're suggesting as alternatives.